tests/input/src/main/scala/example/Miscellaneous.scala

```diff
@@ -4,6 +4,10 @@ class Miscellaneous {
   // backtick identifier
   val `a b` = 42
 
+  // block with only wildcard value
+  def apply(): Unit = {
+    val _ = 42
```

Comment:
Nice job tracking down the issue! I encountered it myself while browsing the fs2 sources.